frontend/src/lobby/LobbyPage.tsx

```diff
@@ -332,22 +320,20 @@ function LobbyPage({ socket }: { socket: any }): JSX.Element {
     const inc = validateIncrement();
     const bs = validateBoardSettings();
     const name = validateName();
-    const eloId = validateEloId();
     updateState((draftState) => {
       draftState.clientRole = RoleEnum.offline;
       // Duration and increment are converted from string to number here.
       draftState.timeControl.duration = dur;
       draftState.timeControl.increment = inc;
       draftState.boardSettings = bs;
       draftState.playerName = name;
-      draftState.eloId = eloId;
+      if (state.idToken === "") draftState.idToken = socket.id;
```

Comment:
Change implementation such that idToken is blank if not logged in -> backend checks that idToken is empty and uses socket.id instead.

Reply:
done